source/docs/transportation/alt-fuel-stations-v1/_record_fields.html.md.erb

```diff
@@ -1326,6 +1334,17 @@
         </table>
       </td>
     </tr>
+    <tr id="ev-charging-units">
+      <th class="doc-parameter-name" scope="row">ev_charging_units</th>
+      <td class="doc-parameter-value">
+        <div class="doc-parameter-value-field">
+          <strong>Type:</strong> array
+        </div>
+      </td>
+      <td class="doc-parameter-description">
+        For electric stations, an array of EV charging unit records. See <a href="#ev-charging-unit-record-fields">EV charging unit record fields</a> for response details.
```

Comment:
```suggestion
        For electric vehicle charging stations, an array of EV charging units. See <a href="#ev-charging-unit-record-fields">EV charging unit record fields</a> for response details.
```
@littleforest – Thanks for asking about this content. These are the text changes I suggest.